host/sgxload.c

```diff
@@ -213,7 +220,11 @@ static void* _allocate_enclave_memory(size_t enclave_size, int fd)
         uint8_t* end = (uint8_t*)base;
 
         if (start != end && munmap(start, (size_t)(end - start)) != 0)
+        {
+            OE_TRACE_ERROR(
+                "Unmap [MPTR...BASE] failed start=0x%x end=0x%x\n", start, end);
```

Comment:
%p is a better format specified for pointers.

Reply:
good point.

fixed.